CHANGELOG.md

```diff
@@ -106,6 +106,8 @@ and this project adheres to
   - [#1580](https://github.com/iovisor/bpftrace/pull/1580)
 - Printing of small integers with `printf`
   - [#1532](https://github.com/iovisor/bpftrace/pull/1532)
+- Add architecture specific testcase support for semantic_analyser
```

Comment:
We only put the enduser facing changes in the changelog, internal test changes are not relevant for them so you can remove this :)

Reply:
Regards.